src/App.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
+
+export const findVisibleMovies = (movies: Movie[], query: string): Movie[] => {
+  return movies.filter(movie => {
+    const { description, title } = movie;
+
+    return description.toLocaleLowerCase()
```

Comment:
maybe let's trim query to get better search results?

Reply:
when I write `query.trim().toLocaleLowerCase()` - I have 1 test falied - **should update search results on type**

Reply:
yeah, guess we have broken tests
that's okay we can fix this